app/util/log.ts

```diff
@@ -15,6 +37,7 @@ function createJsonLogger(): winston.Logger {
       winston.format.timestamp(),
       envNameFormat(),
       winston.format.json(),
+      urlAndMessageBasedFilter(),
```

Comment:
I understand ignoring these when using TEXT_LOGGER (maybe) but what's the motivation for ignoring these in the JSON logger.  Aren't the messages being ignored used for pulling timing metrics?

Reply:
Mostly yes. I should probably remove it from the JSON logger if that has no relevance.

Reply:
Yes, I would remove it from JSON logger, since they're needed for prod there. A couple of the ones being filtered were specifically added to support metrics

Reply:
To clarify: I think removing it from the JSON logger gets at what's needed for this PR. Potentially add a clean-up item to the list for a future sprint for the other one about using log levels or tags